### DISCUSSION TITLE
LSTMATTN과 BERT의 차이를 모르겠어요!

### DISCUSSION THREAD

**Discussion by gonipark** [bcaitech1/p4-dkt-lastffang#19] [Q&A]
lstm에 attention을 추가해준게 결국 transformer인거 아닌가요? 

bert가 바로 transformer를 사용하는 모델인데, 그럼 둘의 차이가 뭘까요?

layer 구성의 차이일까요?

**Comment by newspring97** [comment]
![IMG_0348](https://user-images.githubusercontent.com/26354919/119781655-3bdd4d00-bf06-11eb-89be-9a291e582a14.PNG)
LSTM은 기본적으로 데이터를 순차적으로 입력 받고 각각의 hidden state를 이용해 어텐션을 만들어 최종 결과에 적용하고 Transformer는 처음 encoder block으로 입력을 할 때 데이터를 한 번에 넣어서 encoder block을 지나갑니다!
대신 Transformer에서 입력을 받을 때 한 번에 데이터를 넣는 경우 데이터의 위치정보가 사라질 수 있기 때문에 positional encoding을 먼저 해주고요
